homeassistant/components/cloud/manifest.json

```diff
@@ -1,9 +1,9 @@
 {
   "domain": "cloud",
   "name": "Home Assistant Cloud",
-  "after_dependencies": ["assist_pipeline", "google_assistant", "alexa"],
+  "after_dependencies": ["google_assistant", "alexa"],
   "codeowners": ["@home-assistant/cloud"],
-  "dependencies": ["http", "repairs", "webhook"],
+  "dependencies": ["http", "repairs", "webhook", "assist_pipeline"],
```

Comment:
I do not like this change. We set up cloud very early on because of providing the OAuth2 service. Now it will pull in all of assist_pipeline because `async_migrate_cloud_pipeline_engine`. Need to rethink this.

Reply:
Moved it back.